.evergreen/compile.sh

```diff
@@ -38,6 +39,7 @@ case "$OS" in
       ;;
 
    cygwin*)
+      GENERATOR=${GENERATOR:-"Visual Studio 14 2015 Win64"}
```

Comment:
Since we have two options for generators on Windows, I made this more consistent with `install_c_driver.sh` and made the generator an optional environment variable for this script.

Reply:
Acknowledged